ui/racer/css/_race.scss

```diff
@@ -1,12 +1,19 @@
-$track-height: 25px;
-$car-width: 4rem;
-$car-ratio: 0.328;
-$c-bg-position-y: 2px;
+:root {
+  --track-height: 25px;
```

Comment:
We use three dashes --- as a prefix for non-color css variables. I realize it may seem a little weird but it's needed to sort them above all the theme color variables in browser dev tools (which does lexical sort).